src/components/NewWorkspaceModal.js

```diff
@@ -190,6 +190,7 @@ const NewWorkspaceModal = withDisplayName(
       // the Events.workspaceClone event data.
       return Utils.cond(
         [!!cloneWorkspace && isAzureWorkspace(cloneWorkspace), () => isAzureBillingProject(project)],
+        [disallowAzure, () => !isAzureBillingProject(project)],
```

Comment:
I don't understand this given the condition immediately before.

Reply:
Good point. Workflow import and clone workspace are never going to happen together but I agree we could make this clearer. Perhaps coping the existing pattern and passing this information in as `workflowImport` (in analogy to the existing `cloneWorkspace`) rather than a blanket `disallowAzure` is the right way to go.

Reply:
Alright, I've made this change. Because this now takes precedence over the clone logic, I think my import-or-no-import test case should be testing the switching on and off appropriately, and otherwise the logic is exactly as before. Does this look better?